Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb

```diff
@@ -0,0 +1,1542 @@
+{
```

Comment:
<p>perhaps you can cross-link to the SimulationAPI notebook? Just so the viewer/user can orientate that some of these features being used and explained have also other dedicated notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Added, thanks.</p><p> </p>